### DISCUSSION TITLE
How to propose excaliburjs.com docs corrections?

### DISCUSSION THREAD

**Discussion by ikudrickiy** [excaliburjs/Excalibur#3003] [Q&A] [answered]
Should I create an issue to pull request with docs corrections?
If so, what should I write in it? I can list the mistakes contained in docs but it sounds like doing unnecessary work as the mistakes are explained in commit extended explanations.

And if I shouldn't, how should I mark the pull request?

**Comment by eonarheim** [comment, marked as answer]
@ikudrickiy good question!

For small things the docs there is no need to discuss beforehand in an issue, things like grammar/typos/fixes to examples/new examples can just be PRd.

I'll be sure to update the contributing documents to clarify.

That step is to prevent people from working hard on some feature/bug to Excalibur that we haven't discussed that we can't accept.

**Comment by eonarheim** [comment]
> Should I create an issue to pull request with docs corrections?

Please do! It would be much appreciated 

> If so, what should I write in it? I can list the mistakes contained in docs but it sounds like doing unnecessary work as the mistakes are explained in commit extended explanations.

All the docs are in /site in markdown files .MDX please make your changes there and submit a PR

**Comment by ikudrickiy** [comment]
> > Should I create an issue to pull request with docs corrections?
> 
> Please do! It would be much appreciated

This question was mainly about whether I could skip creating an issue in case of document edits. You answered it above 😉
